routes/ffxiv.py

```
@@ -46,3 +46,54 @@
+
+        response = requests.post(
+            f"{api_url}/wow/shoppinglistx",
+            headers={"Accept": "application/json"},
+            json=json_data,
+        ).json()
+
+        if "data" not in response:
+            logger.error(
+                f"Error no matching data with given inputs {json_data} response {response}"
+            )
+            if NO_RATE_LIMIT:
```

Ruff

Undefined name `NO_RATE_LIMIT`

(F821)